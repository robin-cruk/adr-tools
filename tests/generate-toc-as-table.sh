adr new First Decision
adr new Second Decision
adr link 2 Supersedes 1 "Superseded by"
adr new Third Decision
adr generate toc -s -t
