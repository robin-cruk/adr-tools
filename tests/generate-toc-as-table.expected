adr new First Decision
doc/adr/0001-first-decision.md
adr new Second Decision
doc/adr/0002-second-decision.md
adr link 2 Supersedes 1 "Superseded by"
adr new Third Decision
doc/adr/0003-third-decision.md
adr generate toc -s -t
# Architecture Decision Records

| ADR | Status | Date |
| :--- | :--- | :--- |
|[1. First Decision](0001-first-decision.md)|Accepted<br>Superseded by [2. Second Decision](0002-second-decision.md)|1992-01-12|
|[2. Second Decision](0002-second-decision.md)|Accepted<br>Supersedes [1. First Decision](0001-first-decision.md)|1992-01-12|
|[3. Third Decision](0003-third-decision.md)|Accepted|1992-01-12|
